CMakeLists.txt

```diff
@@ -172,14 +177,9 @@ if(TRAVIS_CI_BUILD)
   add_definitions(-Werror)
 endif()
 
-if(CMAKE_COMPILER_IS_GNUCC)
-  check_c_compiler_flag(-Og HAS_OG_FLAG)
-else()
-  set(HAS_OG_FLAG 0)
-endif()
```

Comment:
@jszakmeister Did this intentionally: https://github.com/neovim/neovim/pull/3137#discussion_r38042611